docs/docs-content/security-bulletins/cve-reports.md

```diff
@@ -32,6 +32,35 @@ _Are there any links users can visit to find out more?_
 
 -->
 
+## April 2, 2024 - CVE-2024-3094 Malicious code was discovered in the upstream tarballs of xz - 10 CVSS
+
+Malicious code was discovered in the upstream tarballs of xz, starting with version 5.6.0. Through a series of complex 
+obfuscations, the liblzma build process extracts a prebuilt object file from a disguised test file existing in the 
+source code, which is then used to modify specific functions in the liblzma code. 
+This results in a modified liblzma library that can be used by any software linked against this library, intercepting 
+and modifying the data interaction with this library.
+
+#### Impact
+
+No impact. Palette uses Ubuntu 22.04 OS images while VerteX uses Ubuntu 20.04 OS images for Enterprise, 
+Private Cloud Gateway and Tenant clusters. Both Ubuntu 22.04 and Ubuntu 20.04 images uses an older non-impacted 
```

Comment:
Can we share what version, just so that's more clear?

Reply:
Installed version may differ depending on the image. For example, OS image from 3.4 or 4.0 will have different version of xz-utils than the ones from 4.2 or 4.3.